### PR TITLE
docs: Fix typo and add contribute section

### PR DESCRIPTION
**Pull request by Xuanwo**

### CODE REVIEW

**Comment by Xuanwo** [comment]
cc @JingsongLi for a review.

File: crates/paimon/Cargo.toml [hunk 1]
```diff
@@ -16,14 +16,15 @@
 # under the License.
 
 [package]
-categories = ["command-line-utilities"]
-description = "The rust implementation of paimon"
+categories = ["database"]
+description = "The rust implementation of Apache Paimon"
```
Comment (QuakeWang) [inline crates/paimon/Cargo.toml:20]:
```suggestion
description = "The Rust implementation of Apache Paimon"
```
Do initial letters need to be capitalized?

Comment (Xuanwo) [reply crates/paimon/Cargo.toml:20]: Both look to me.

File: README.md [hunk 2]
```diff
@@ -19,20 +19,25 @@
 
 # Apache Paimon Rust &emsp; [![Build Status]][actions] [![Latest Version]][crates.io]
 
-[Build Status]: https://img.shields.io/github/actions/workflow/status/apache-rust/paimon/ci.yml
-[actions]: https://github.com/apache-rust/paimon/actions?query=branch%3Amain
+[Build Status]: https://img.shields.io/github/actions/workflow/status/apache/paimon-rust/ci.yml
+[actions]: https://github.com/apache/paimon-rust/actions?query=branch%3Amain
```
Comment (SteNicholas) [inline README.md:23]:
```suggestion
[Actions]: https://github.com/apache/paimon-rust/actions?query=branch%3Amain
```

File: README.md [hunk 3]
```diff
@@ -19,20 +19,25 @@
...
 [Latest Version]: https://img.shields.io/crates/v/paimon.svg
 [crates.io]: https://crates.io/crates/paimon
 
 The rust implementation of Apache Paimon. 
 
-## Quick Start
+## Progress
```
Comment (SteNicholas) [inline README.md:29]:
```suggestion
## Issue Tracker
```

File: README.md [hunk 4]
```diff
@@ -19,20 +19,25 @@
...
 
-Working in progress
+See [Tracking issues of paimon rust 0.1.0 release](https://github.com/apache/paimon-rust/issues/3)
```
Comment (SteNicholas) [inline README.md:31]:
```suggestion
See [Tracking issues of 0.1.0 version for Paimon Rust](https://github.com/apache/paimon-rust/issues/3)
```

File: README.md [hunk 5]
```diff
@@ -19,20 +19,25 @@
...
 
 ## Contributing
 
-Check out the [CONTRIBUTING.md](./CONTRIBUTING.md) guide for more details on getting started with contributing to this project.
+Apache Paimon Rust is under active development. We are always open to people who want to use it or contribute to it. Here are some ways to go.
```
Comment (SteNicholas) [inline README.md:35]:
```suggestion
Apache Paimon Rust is an exciting project currently under active development. Whether you're looking to use it in your projects or contribute to its growth, there are several ways you can get involved:
```

File: README.md [hunk 6]
```diff
@@ -19,20 +19,25 @@
...
+
+- Start with [Contributing Guide](CONTRIBUTING.md).
+- Submit [Issues](https://github.com/apache/paimon-rust/issues/new) for bug report or feature requests.
```
Comment (SteNicholas) [inline README.md:38]:
```suggestion
- Create new [Issue](https://github.com/apache/paimon-rust/issues/new) for bug reportor or feature request.
```

File: README.md [hunk 7]
```diff
@@ -19,20 +19,25 @@
...
+- Discuss at [dev mailing list](mailto:dev@paimon.apache.org) ([subscribe](<mailto:dev-subscribe@paimon.apache.org?subject=(send%20this%20email%20to%20subscribe)>) / [unsubscribe](<mailto:dev-unsubscribe@paimon.apache.org?subject=(send%20this%20email%20to%20unsubscribe)>) / [archives](https://lists.apache.org/list.html?dev@paimon.apache.org))
```
Comment (SteNicholas) [inline README.md:39]:
```suggestion
- Start discussion thread at [dev mailing list](mailto:dev@paimon.apache.org) ([subscribe](<mailto:dev-subscribe@paimon.apache.org?subject=(send%20this%20email%20to%20subscribe)>) / [unsubscribe](<mailto:dev-unsubscribe@paimon.apache.org?subject=(send%20this%20email%20to%20unsubscribe)>) / [archives](https://lists.apache.org/list.html?dev@paimon.apache.org))
```

File: README.md [hunk 8]
```diff
@@ -19,20 +19,25 @@
...
 
-Working in progress
+See [Tracking issues of paimon rust 0.1.0 release](https://github.com/apache/paimon-rust/issues/3)
 
 ## Contributing
 
-Check out the [CONTRIBUTING.md](./CONTRIBUTING.md) guide for more details on getting started with contributing to this project.
+Apache Paimon Rust is under active development. We are always open to people who want to use it or contribute to it. Here are some ways to go.
+
+- Start with [Contributing Guide](CONTRIBUTING.md).
```
Comment (SteNicholas) [inline README.md:37]:
```suggestion
- Follow the [Contributing Guide](CONTRIBUTING.md) to contribute.
```

File: README.md (hunk 2 above)
Comment (Xuanwo) [reply README.md:23]: Hi, this label served as markdown link and not readable by users. If we do want to change this, we should change the reference too.

Comment (Xuanwo) [reply README.md:23]: I will ignore this suggestion. Feel free to reopen it if you feel like this change does needed.

**Review by SteNicholas — APPROVED** [review]

**Comment by Xuanwo** [comment]
All comments have been addressed, PTAL @SteNicholas

**Pull request merged**